src/sentry/notifications/helpers.py

```diff
@@ -472,3 +473,61 @@ def get_most_specific_notification_setting_value(
         )
         or _get_notification_setting_default(ExternalProviders.EMAIL, type)
     )
+
+
+def are_there_any_settings(
```

Comment:
If this is supposed to be a library function, can you add docstrings?

Reply:
What do you think of calling this `has_settings`?